src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -40,35 +40,29 @@
 
     public ShooterSubsystem shooterSubsystem;
 
-    public Subsystems(Hardware h) {
-        boolean comp = Robot.getInstance().isCompetition();
+    private Subsystems() {
+        boolean comp = Robot.instance.isCompetition();
```

Comment:
Should instead pass in the robot type to the constructor.  And it really should be an `enum` and not a `boolean`

Reply:
Likely also should pass in the `physicsSim` so it can be passed down to each individual subsystem, which can then add their hardware to it (and encapsulate the simulation of their hardware vs. having it far away in the `Robot` class.